tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -746,3 +748,134 @@ def test_02_multiattach_volumes(self):
+                    vol_type_name = vol
+
+            if (vol_type_id == -1):
+                raise Exception("No multiattach volume found to create multiattach volume. Test cannot be continued")
+
+            # Now create volume with derived volume type id...
+            self.volume_id = self.create_volume(
+                volume_type_id=vol_type_id, size=6, volume_cleanup=False)
+
+            LOG.debug("Volume ID: " + str(self.volume_id))
+
+            self.volumes = []
+            self.volumes.append(self.volume_id)
+            # Attach volume to vm...
+            api_version.COMPUTE_MICROVERSION = '2.60'
+            self.attach_volume(self.volume_id, self.vm_id_1, attach_cleanup=False)
+            self.attach_volume(self.volume_id, self.vm_id_2, attach_cleanup=False)
+            LOG.debug("Multiattach Volume attached to vm: " + str(self.vm_id_1) + " and " + str(self.vm_id_2))
+            api_version.COMPUTE_MICROVERSION = None
+
+            vol_vm_1 = self.get_attached_volumes(self.vm_id_1)
+            vol_vm_2 = self.get_attached_volumes(self.vm_id_2)
+            LOG.debug("Voulme o VM 1: " + str(vol_vm_1) + " on VM 2:" + str(vol_vm_2))
+            if vol_vm_1 == vol_vm_2:
+                reporting.add_test_step("Attached Multiattach volume to both Instances", tvaultconf.PASS)
+                reporting.set_test_script_status(tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Attached Multiattach volume to both Instances", tvaultconf.FAIL)
+                reporting.set_test_script_status(tvaultconf.FAIL)
+                raise Exception("Multiattach volume failed to attach existing instance")
+
+            # Create workload with API
+            try:
+                self.wid = self.workload_create([self.vm_id_1, self.vm_id_2],
+                                                workload_type=tvaultconf.parallel,
+                                                jobschedule=self.schedule,
+                                                workload_name='Workload-1',
+                                                description='New Test')
+                LOG.debug("Workload ID: " + str(self.wid))
+            except Exception as e:
+                LOG.error(f"Exception: {e}")
+                raise Exception("Create workload with image booted vm")
+            if (self.wid is not None):
+                self.wait_for_workload_tobe_available(self.wid)
+                self.workload_status = self.getWorkloadStatus(self.wid)
+                if (self.workload_status == "available"):
+                    reporting.add_test_step("Create workload", tvaultconf.PASS)
+                else:
+                    raise Exception("Create workload failed")
+            else:
+                raise Exception("Create workload failed")
+
+            LOG.debug("Sleeping till snapshots get completed")
+            time.sleep((int(tvaultconf.interval.split(' ')[
```

Comment:
What is the max sleep time here?

Reply:
This waits for scheduler to trigger snapshot and then snapshot to complete. Instead of scheduler triggered snapshot, trigger a snapshot explicitly using API.

Reply:
Total sleep time is 4hours 10 mins to create and wait to complete 4 Snapshots

Reply:
removed